python/lsst/ctrl/execute/slurmPlugin.py

```diff
@@ -272,7 +278,7 @@ def createAllocationFile(self, input):
         os.chmod(outfile, 0o755)
         return outfile
 
-    def glideinsFromJobPressure(self, generatedSlurmFile):
+    def glideinsFromJobPressure(self, platformPkgDir):
         """Determine and submit the glideins needed from job pressure"""
```

Comment:
Since updating parameter name, could update function docstring to have Parameters section.

Reply:
Added

        """Determine and submit the glideins needed from job pressure

        Parameters
        ----------
        platformPkgDir : `str`
            path to the ctrl_platform package being used
        """